CHARTER.md

```diff
@@ -1,59 +1,61 @@
+If you wish to join the ART, please submit a pull request adding yourself to the [`ART_MEMBERS.md`](ART_MEMBERS.md) file. Please describe your use case in the pull request comments; you do not need to mention any company names or details. Joining is finalized upon two pull request approvals by active chairs and the merging of the pull request.
 
-Additionally, as stated in the [Arcalot Code of Conduct](https://github.com/arcalot/.github/blob/main/CODE_OF_CONDUCT.md), chairs can remove an ART member if their behavior is clearly disruptive to the community.
+Community meetings are not compulsory as we want to include people in all stages of life and across all timezones. We therefore refrain from any synchronous or undocumented decision making.
+
+As stated in the [Arcalot Code of Conduct](https://github.com/arcalot/.github/blob/main/CODE_OF_CONDUCT.md), chairs can remove any ART member if their behavior is clearly disruptive to the community.
 
 
 ### Chair Participation
 
-Chairs are expected to remain actively involved in the community. An Arcalot Round Table chair that does not participate in round table matters (voting) *without reason and notice* for 3 times in a row leaves the chairs automatically as we can assume they are not interested in the responsibilities of the chair position anymore.
+Chairs are expected to remain actively involved in the community. A chair who does not participate in ART voting *without reason and notice* three times in a row forfeits the position and becomes a regular ART member automatically.
 
 
-## Decision Proposals
+## Proposals
 
-Decisions can be started *only asychronously* in the [ART repository pull requests](https://github.com/arcalot/arcalot-round-table/pulls) via submitting a pull request with a new file to [`art-decisions/proposals`](art-decisions/proposals). You can use the included [template file](art-decisions/proposals/template.md) as a guideline. Decision proposals can only be submitted by ART members. If you would like to encourage discussion around a decision before any voting starts, please submit a draft PR instead and mark it ready for review once you are comfortable that you will not add any changes to your proposal.
+Proposals may only be made asychronously via the [ART repository pull requests](https://github.com/arcalot/arcalot-round-table/pulls) by submitting a pull request with a new file to [`art-decisions/proposals`](art-decisions/proposals). The provided [template file](art-decisions/proposals/template.md) may be used as a guideline for new proposals. Proposals may only be submitted by current ART members. If you would like to encourage discussion around a proposal before any voting starts, please submit a draft PR instead and mark it ready for review once you are comfortable that you will not make any changes to your proposal.
```

Comment:
This actually sounds awkward ... "via the [Pull requests] by submitting a pull request". Instead, maybe "Proposals may be made by submitting a new file under [art-decisions/proposals]. You can view all open pull requests at [ART repository pull requests]."